internal/signalfx-agent/pkg/monitors/processlist/processlist_linux.go

```diff
@@ -68,18 +69,16 @@ func ProcessList(conf *Config, cache *osCache, logger logrus.FieldLogger) ([]*To
 
 		username := ""
 		uid := status.UIDs[0]
-		if uid != "" {
```

Comment:
I wonder if there is a possibility for uids to be empty? not sure why we had this check

Reply:
_Whoops, didn't refresh before leaving my comment._ 👍